examples/Quasiconvex Optimization by Bisection.ipynb

```diff
@@ -0,0 +1,197 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Solve the Quasiconvex problem by Convex feasibility and Bisection Search\n",
+    "A quasiconvex problem is defined as - \n",
+    "\n",
+    "\\begin{array}\n",
+    "   \\mbox{minimize}    & f_0(x)\\\\\n",
+    "   \\mbox{subject to}  & f_i(x) \\leq 0  \\\\ \n",
+    "   & Ax = b\n",
+    "\\end{array}\n",
+    "\n",
+    "where $f_0 : \\mathbb{R}^n \\rightarrow \\mathbb{R}$ is Quasiconvex and the remaining functions are convex. \n",
```

Comment:
I'd suggest starting by defining what a quasiconvex function is, and explaining why these problems are *not* (necessarily) convex.